tests/integration_tests/test_pytorch_lightning.py

```diff
@@ -45,6 +45,8 @@ def validation_step(  # type: ignore
         return {"validation_accuracy": accuracy}
 
     def validation_epoch_end(self, outputs: List[Dict[str, torch.Tensor]]) -> None:
+        if not len(outputs):
+            return
```

Comment:
For DDP test, this value is empty. To avoid zero-division of the following `accuracy`'s line, I've introduced this condition.

Reply:
nice catch 👍